ToDo.Core/ViewModels/ToDoViewModel.cs

```diff
@@ -111,35 +199,72 @@ internal async Task AddNewTodo()
             await _navigationService.Navigate<AddViewModel>();
         }
 
-        internal async Task RemoveTodo(Models.ToDo toDoItem)
+        internal async Task RemoveTodo(VisualElement frontSide)
         {
-            var confirmed = await _userDialogService.ConfirmAsync(new ConfirmConfig()
-            {
-                Message = "Are you sure you want to delete this item?",
-                OkText = "Delete",
-                CancelText = "Cancel"
-            });
+            //var confirmed = await _userDialogService.ConfirmAsync(new ConfirmConfig()
+            //{
+            //    Message = "Are you sure you want to delete this item?",
+            //    OkText = "Delete",
+            //    CancelText = "Cancel"
+            //});
 
-            if (confirmed)
-            {
-                ToDos.Remove(toDoItem);
+            //if (confirmed)
+            //{
+            //    ToDos.Remove(SelectedTodo);
+
+            //    await _todoService.DeleteTodo(SelectedTodo);
+            //}
+
+            var frontSideToFlip = frontSide.FindByName<Grid>("FrontSideToFlip");
```

Comment:
This is actually pretty bad practice:
Your ViewModel **must **never**** have a reference to its view.
Otherwise, how do you test this? (Can you show me a UT for this? ;-))

Therefore, it is **much** better to have such *view modification code* in the code behind.
Remember:
- code that changes the ui only (handles animation, etc) can go to the code behind file
- code that implements some logic ("business" logic) **must** go into the viewmodel in order to be testable!